python/torch_mlir_e2e_test/test_suite/slice_like.py

```diff
@@ -677,6 +677,78 @@ def SliceCopyNonZeroDim_Module_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class SelectFloatCopyNonZeroDim_Module(torch.nn.Module):
```

Comment:
Can you also add a test for the non-zero case? Something like `torch.ops.aten.select(input, dim=1, index=3)`